var/spack/repos/builtin/packages/py-spykeutils/package.py

```diff
@@ -0,0 +1,39 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class PySpykeutils(PythonPackage):
+    """Utilities for analyzing electrophysiological data"""
+
+    homepage = "https://github.com/rproepp/spykeutils"
+    url      = "https://pypi.io/packages/source/s/spykeutils/spykeutils-0.4.3.tar.gz"
+
+    version('0.4.3', 'cefe4c48ebfdb9bac7a6cbfaf49dd485')
+
+    depends_on('py-setuptools',      type='build')
+    depends_on('py-scipy',           type=('build', 'run'))
+    depends_on('py-quantities',      type=('build', 'run'))
+    depends_on('py-neo@0.2.1:0.4',   type=('build', 'run'))
```

Comment:
Spack version ranges are inclusive, so `0.2.1:0.4` means 0.2.1 <= x <= 0.4. According to the package, this needs 0.2.1 <= x < 0.4. You'll have to use `0.2:0.3.9` or something like that.